woocommerce/payment-gateway/class-sv-wc-payment-gateway.php

```diff
@@ -3134,19 +3134,24 @@ protected function add_authorization_charge_form_fields( $form_fields ) {
 		}
 
 		// get a list of the "paid" status names
-		$paid_statuses = array_map( 'wc_get_order_status_name', SV_WC_Plugin_Compatibility::wc_get_is_paid_statuses() );
+		$paid_statuses = array_map( 'wc_get_order_status_name', (array) SV_WC_Plugin_Compatibility::wc_get_is_paid_statuses() );
 
 		// do some oxford comma magic
-		$last_status = array_pop( $paid_statuses );
-		array_push( $paid_statuses, "or {$last_status}" );
-		$separator = count( $last_status ) < 3 ? ' ' : ', ';
+		if ( count( $paid_statuses ) > 1 ) {
+
+			$last_status = array_pop( $paid_statuses );
+
+			array_push( $paid_statuses, "or {$last_status}" );
```

Comment:
This doesn't seem to be translatable though.

By the way Memberships had since v1.0.0 an utility function called `wc_memberships_list_items()` to do what you're doing here, but makes the 'or' conjunction customizable (e.g. use `and` instead of `or`) and translatable.

You might consider importing the function `wc_memberships_list_items()` in framework as I've seen a few uses of that in WC plugins.

Reply:
I'd stick with this format and make a FW helper if anything, as the one in Memberships doesn't use an Oxford comma though it should 😸then migrate any other usages to the FW helper

Reply:
added an issue for the helper: https://github.com/skyverge/wc-plugin-framework/issues/279